packages/babel-generator/test/index.js

```diff
@@ -740,15 +740,19 @@ describe("programmatic generation", function () {
       expect(output).toBe(`"\\u8868\\u683C_\\u526F\\u672C"`);
     });
 
-    it("default", () => {
-      const output = generate(string).code;
+    if (process.env.BABEL_8_BREAKING) {
+      it("default in Babel 8", () => {
```

Comment:
We should name this `default` and the other one `default in Babel 7`, since we'll eventually only keep the Babel 8 test in the future.

This is similar to how Babel 7-specific fixtures are named `something-something-babel-7`, while Babel 8 fixtures are just `something-something`.